_studio/mfx_lib/encode_hw/mpeg2/src/mfx_mpeg2_encode_utils_hw.cpp

```diff
@@ -1178,24 +1178,20 @@ namespace MPEG2EncoderHW
         {
             bInvalid = true;
         }
-        if ((pFrameInfo->Width  & 15) !=0)
+        if ((pFrameInfo->Width & 15) != 0)
```

Comment:
IMO, it makes sense to add utility function like `bool aligned(value, alignment)` for places like this

Reply:
agree, but this would be the topic for different PR